geoips/plugins/modules/readers/seviri_hrit.py

```diff
@@ -19,16 +19,22 @@
 import os
 import logging
 import numpy as np
-from geoips.plugins.modules.readers.utils.hrit_reader import HritFile, HritError
+from struct import error as structerror
 
-# Installed Libraries
+# Third-Party Libraries
+import xarray
 
-# GeoIPS Libraries
```

Comment:
Why this change? See my above comment for my opinion

Reply:
This is addressed in [this](https://github.com/NRLMMD-GEOIPS/geoips/commit/b8155a374888d6a5e646107b11a22369aaa88384) commit